app/helpers/status_helper.rb

```diff
@@ -21,6 +21,10 @@ def status_label(key)
 
   def status_panel(deploy)
     content = h deploy.summary
+    if deploy.active?
+      content << content_tag(:ul)
```

Comment:
a lone ul and then text ... how about:

`content_tag(:ul, content_tag(:li, deploy.summary_for_process))` ?